hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassStatement.java

```diff
@@ -312,9 +312,10 @@
 
         SELECT_TAGS_TRIGGERS_ALL_BY_NAME = "SELECT tenantId, triggers "
                 + "FROM " + keyspace + ".tags_triggers "
-                + "WHERE name = ? ";
+                + "WHERE name = ? "
+                + "ALLOW FILTERING";
```

Comment:
Right, adding tenantId would remove the need for "ALLOW FILTERING" but we don't want the tenantId in the where clause as this is an "ALL" query, specifically for cross-tenant fetch.  It is not expected to be very performant.  You are right that if we had a way to get every tenant we could then run multiple queries in parallel, which may be more performant.   Do you want to change the approach for this PR?